src/app/static/vue.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+    lintOnSave: false,
+    outputDir: "../server/public/compiled"
```

Comment:
I am trying to find out how a file `./src/app/server/public/compiled/demo.html` ends up being created (related to the stochastic app) but it's not obvious to me. More generally there's quite a bit of stuff that ends up in that directory - does it all want being made public?

Reply:
Oh, that's just output by the vue cli build, you're right, we don't need that. Rather than the build going directly to the public dir, I'll make it build into static, and then just copy the bits we need.